ViewControllers/PlayingNowCollectionViewController.swift

```diff
@@ -53,26 +56,17 @@ class PlayingNowCollectionViewController: UICollectionViewController, UICollecti
     override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: PlayingNowCollectionViewCell.reuseIdentifier, for: indexPath) as? PlayingNowCollectionViewCell else { fatalError() }
         
-        let movieName = movies[indexPath.item].title
-        let reviewsScore = "\(movies[indexPath.item].voteAverage)%"
-        let path = movies[indexPath.item].posterPath
-        if let url = movieDataService.getMoviePosterURL(posterPath: path) {
-            cell.configure(imageURL: url, name: movieName, duration: "0h0", reviewsScore: reviewsScore)
+        let movie = movies[indexPath.item]
+        let movieName = movie.title
+        let reviewsScore = movie.voteAverage
+        let popularity = movie.popularity
+        if let path = movie.posterPath {
+            cell.configure(imageURL: path, name: movieName, reviewsScore: reviewsScore, popularity: popularity)
```

Comment:
Let's make the `cell.configure` method accept an optional imageURL, so we will always call it even when there is no poster path. The way we have here, in case a move has no poster image it will show as a completely empty cell even though we have other data such name, popularity, etc.